src/doc/man/cargo-install.md

```diff
@@ -69,6 +69,14 @@ not start publishing `Cargo.lock` files until version 1.37, which means
 packages published with prior versions will not have a `Cargo.lock` file
 available.
 
+This command operates on system or user level, not project level.
```

Comment:
Not really orthogonal to the issue, just feel like the description becomes way too long at this moment. Could we have h3 headers for this paragraph, and maybe one for `Cargo.lock paragraph? I am think about something like “Dealing with lockfile" and “Configuration discovery”.